src/serlio/modifiers/RuleAttributes.cpp

```diff
@@ -57,6 +57,27 @@ std::wstring getNiceName(const std::wstring& fqAttrName) {
 
 } // namespace
 
+std::map<std::wstring,int> getImportOrderMap(const prt::RuleFileInfo* ruleFileInfo) {
+	std::map<std::wstring, int> importOrderMap;
+	int importOrder = 0;
+	for (size_t i = 0; i < ruleFileInfo->getNumAnnotations(); i++) {
+		const prt::Annotation* an = ruleFileInfo->getAnnotation(i);
+		const wchar_t* anName = an->getName();
+		if (!(std::wcscmp(anName, ANNOT_IMPORTS))) {
+			for (int argIdx = 0; argIdx < an->getNumArguments(); argIdx++) {
+				if (an->getArgument(argIdx)->getType() == prt::AAT_STR) {
+					const wchar_t* anKey = an->getArgument(argIdx)->getKey();
+					if(!(std::wcscmp(anKey, ANNOT_IMPORTS_KEY))) {
+						std::wstring importRule = an->getArgument(argIdx)->getStr();
```

Comment:
`getStr` returns a pointer and should be null-checked before constructing a `std::wstring`